compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -60,7 +61,19 @@ pub struct CompileTimeInterpreter<'mir, 'tcx> {
     pub(super) check_alignment: CheckAlignment,
 
     /// Used to prevent reads from a static's base allocation, as that may allow for self-initialization.
```

Comment:
```suggestion
    /// If `Some`, we are evaluating the initializer of the static with the given `LocalDefId`,
    /// storing the result in the given `AllocId`.
    /// Used to prevent reads from a static's base allocation, as that may allow for self-initialization loops.
```